package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This is wrong... you're doing Java, so you should NOT need a package.json. You probably did something like `yarn install postgres`